pkgs/development/python-modules/geopandas/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi,
+  pandas, shapely, fiona, descartes, pyproj }:
+
+buildPythonPackage rec {
+  pname = "geopandas";
+  version = "0.3.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "02wj58aqyq1nr0axz2vci72zvpkmalrj570lrndqqvai7qmb6fz6";
+  };
+
+  propagatedBuildInputs = [
+    pandas
+    shapely
+    fiona
+    descartes
+    pyproj
+  ];
+
+  meta = {
```

Comment:
Did you want to add yourself as maintainer for all these packages?

Reply:
Uh yeah. I'll do that!